src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeFetchers.scala

```diff
@@ -545,6 +545,50 @@ trait BlocksDataFetchers {
     }
   }
 
+  /** a fetcher of baker rolls for blocks */
```

Comment:
This might be unnecessary, if we get data from the stored db-records.
Or we might want to convert the existing one to fetch from hashes of blocks, like here.

See also the comment on use site code.

Reply:
removed